entrypoint.sh

```diff
@@ -183,6 +183,17 @@ function installThemesFromSources() {
   fi
 }
 
+function manageWPCron() {
+  echo 'control WP cron status' 
+  [ ! -z "$WORDPRESS_DISABLE_CRON" ] && {
```

Comment:
```suggestion
  [ ! -z "${WORDPRESS_DISABLE_CRON:-}" ] && {
```
I'm not confident with shebang restriction. We may need to pass it for the first check.